server/game/gameSystems/SearchDeckSystem.ts

```diff
@@ -248,15 +249,15 @@ export class SearchDeckSystem extends PlayerTargetSystem<ISearchDeckProperties>
         }
     }
 
-    private handleDoneMessage(properties: ISearchDeckProperties, context: AbilityContext, event: any, selectedCards: Set<Card>): void {
+    private searchCompleteMessageHandler(properties: ISearchDeckProperties, context: AbilityContext, event: any, selectedCards: Set<Card>): void {
```

Comment:
Sorry, just now catching this - maybe this should be named `searchCompleteHandler` or something since the name implies it's just doing a message, but it is also potentially activating other handlers. Or just fold the functionality directly into `onSearchComplete`.